sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java

```diff
@@ -555,6 +563,20 @@ private List<UnresolvedExpression> multiFieldRelevanceArguments(
     return builder.build();
   }
 
+  private List<UnresolvedExpression> getFormatFunctionArguments(
+      GetFormatFunctionCallContext ctx) {
+    // first argument is defaulted to string values
```

Comment:
Don't think this comment applies to your `getFormatFunctionArguments` method.

Reply:
It sort of does. I changed the comment to specify that the first argument gets changed because it gets converted to a string. It needs to do this since there is no type that can be used for the signature since the first argument is one of `DATE`, `DATETIME`, `TIME`, `TIMESTAMP` (the actual word, not an argument of that type). I'll change the comment to be clearer.

Reply:
Ignore this. Removed comment due to change in implementation.